main.py

```diff
@@ -19,15 +20,28 @@
     screen.fill("red")
 
     # RENDER YOUR GAME HERE
-    snake = pygame.Rect(640, 360, 20, 20)
-    pygame.draw.rect(surface=screen, color="black", rect=snake)
-    # snake = Snake()
-    # pygame.draw.rect(surface=screen, color="black", rect=pygame.Rect(
-    #     snake.pos_x,
-    #     snake.pos_y,
-    #     snake.width,
-    #     snake.height
-    # ))
+
+    pygame.draw.rect(surface=screen, color="black", rect=pygame.Rect(
+        snake.pos_x,
+        snake.pos_y,
+        snake.width,
+        snake.height
+    ))
+    
+    #implement input here
+    key_input = pygame.key.get_pressed()
```

Comment:
let's add more inputs to each direction

Moving up: w, up arrow, k
Moving down: s, down arrow, j
Moving left: a, left arrow, h
Moving right: d, right arrow, l

also please remove the print comments